pysrc/models/column_statistics.py

```diff
@@ -0,0 +1,17 @@
+from dataclasses import dataclass
+from dataclasses_json import dataclass_json
+
+from .column_information import ColumnInformation
+
+@dataclass_json
+@dataclass(frozen=True)
+class ColumnStatistic:
+    
+    column_Information: ColumnInformation
```

Comment:
```suggestion
class ColumnStatistic:
    column_information: ColumnInformation
```

Remove empty line and fix the casing.